data_generators/random_match.py

```diff
@@ -0,0 +1,29 @@
+from random import sample
+from typing import Dict, List
+
+from pandas import DataFrame
+
+from app.data import MongoDB
+
+data: List[Dict] = MongoDB("UnderdogDevs").read("Mentors")
+
+
+def rand_lib(n_matches: int) -> List[str]:
+    return [match["user_id"] for match in sample(data, k=n_matches)]
+
+
+def pandas_lib(n_matches: int) -> List[str]:
+    return DataFrame(data).sample(n_matches)["user_id"].to_list()
+
+
+if __name__ == '__main__':
```

Comment:
Do we need `if __name__ == '__main':` code block in our data generators and matching? I think the code could be handed over to members working on creating tests. I believe there should be a main script call for the data generation, but I think this doesn't do that.